### PR TITLE
🎷🐋

### PR DESCRIPTION
**Pull request by ThoseGrapefruits**
```
D A T A /
          T R A N S F E R /
                            I N I T I A L I S E D
```

Still have a bunch of print statements, but they're good to have in there to understand what's going on when you run it. This finishes the backend part of the frontend work I was doing earlier, meaning that the backend now supplies a JSON representation of a face when a `get` request is sent to `/faceGraph` and the frontend renders it.

Root Face

### CODE REVIEW

**Comment by ThoseGrapefruits** [comment]
Merging so I can move onto bigger and brighter things

File: src/Data.hs [hunk 1]
```diff
@@ -46,20 +47,70 @@ queryMovie title = do result <- head <$> queryP cypher params
...
+                      liftIO . putStrLn $ "// relations: "
+                      liftIO . print $ relations
                       return $ MGraph (actors <> movies) relations
   where cypher = "MATCH (m:Movie)<-[:ACTED_IN]-(a:Person) " <>
-                 "RETURN m.title as movie, collect(a.name) as cast " <>
+                 "RETURN m.title as movie, collect(a) as cast " <>
                  "LIMIT {limit}"
         params = fromList [("limit", I limit)]
 
+-- |Returns face with all it's points
```
Comment (dackJavies) [inline src/Data.hs:76]: *its

Comment (ThoseGrapefruits) [reply src/Data.hs:76]: what happens when you copy code from others without reading it

Comment (dackJavies) [reply src/Data.hs:76]: those silly folk's

**Pull request merged**